evalml/pipelines/components/transformers/transformer.py

```diff
@@ -55,9 +55,9 @@ def fit_transform(self, X, y=None):
         Raises:
             MethodPropertyNotFoundError: If transformer does not have a transform method or a component_obj that implements transform.
         """
-        X_ww = infer_feature_types(X)
+        X_ww = infer_feature_types(X, ignore_nullable_types=True)
```

Comment:
I think I prefer we implement fit_transform in ReplaceNullableTypes component until we're done with this epic as opposed to modifying the parent class code? This assumes the nullable types have been replaced prior to being passed to fit_transform which isn't true yet.

Reply:
Done!